provider/cmd/pulumi-resource-kubernetes-cert-manager/schema.json

```diff
@@ -72,6 +72,7 @@
                     "$ref": "#/types/kubernetes-cert-manager:index:CertManagerGlobal"
                 },
                 "helmOptions": {
+                    "plain": true,
```

Comment:
This makes sense because helmbase doesn't provide a `ReleaseTypeInput` type to accept an inputty value.  Hence we see the error message:
```
copying input "helmOptions": expected destination type to implement pulumi.Input or pulumi.Output, 
got helmbase.ReleaseType
```